python/llm/src/ipex_llm/utils/ipex_importer.py

```diff
@@ -63,10 +74,11 @@ def import_ipex(self):
             if 'ipex' in sys.modules or 'intel_extension_for_pytorch' in sys.modules:
                 logging.error("ipex_llm will automatically import intel_extension_for_pytorch.")
```

Comment:
The error message should be "Please import ipex_llm before importing intel_extension_for_pytorch"